packages/@aws-cdk/aws-pipes-sources-alpha/test/integ.dynamodb.ts

```diff
@@ -1,4 +1,6 @@
-import { ITarget, Pipe, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { Pipe } from '@aws-cdk/aws-pipes-alpha';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Do we need this es-lint comment?

Reply:
My linter was complaining about it..

Reply:
I added `@aws-cdk/aws-pipes-targets-alpha` to `devDependencies` to fix this.